nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/distribution/SimplifiedServerInfoWriter.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour Al Kotob
+ */
+
+package org.nuxeo.rest.management.distribution;
+
+import static org.nuxeo.ecm.core.io.registry.reflect.Instantiations.SINGLETON;
+import static org.nuxeo.ecm.core.io.registry.reflect.Priorities.REFERENCE;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import org.nuxeo.common.Environment;
+import org.nuxeo.ecm.admin.runtime.PlatformVersionHelper;
+import org.nuxeo.ecm.admin.runtime.SimplifiedBundleInfo;
+import org.nuxeo.ecm.admin.runtime.SimplifiedServerInfo;
+import org.nuxeo.ecm.core.io.marshallers.json.AbstractJsonWriter;
+import org.nuxeo.ecm.core.io.registry.reflect.Setup;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+
+@Setup(mode = SINGLETON, priority = REFERENCE)
+public class SimplifiedServerInfoWriter extends AbstractJsonWriter<SimplifiedServerInfo> {
+
+    @Override
+    public void write(SimplifiedServerInfo entity, JsonGenerator jg) throws IOException {
+        Collections.sort(entity.getBundleInfos());
```

Comment:
No point sorting I believe, it is already done by https://github.com/nuxeo/nuxeo/blob/10.10/nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/ecm/admin/runtime/RuntimeInstrospection.java#L75